### ISSUE TITLE
Feature Request: Find within projects

### ISSUE THREAD

**Issue by hoppfrosch** [Komodo/KomodoEdit#2282]
With "Find" you can search within multiple scopes:

- Current File
-  Selected Text
-  Opened Files
-  Project
-  Files

On the other side you have a list of projects available within a project pane.

It would be nice to have a **"find within projects**" .... (all projects from the projects pane)

_Background_: I'm a perl developer and I'm working on several perl modules and a main application at once (most modules are within their own komodo project - the main app is in a own komodo project as well). So if I'm searching for example for a function-definition of a function call I often have to switch to a different project and to search within the other project and switch back to the original project and search for the original position again ....

**Comment by Naatan** [comment]
Thanks @hoppfrosch, we'll keep it in consideration for future planning.

**Comment by cgchoffman** [comment]
Good idea @hoppfrosch.  In the mean time you can select multiple folders (eg. project dirs) when you're searching in the Files scope by clicking the `...` button to the right of the Directories field.
![capture](https://cloud.githubusercontent.com/assets/655154/22343972/08cf684e-e3af-11e6-926a-284c9b47205a.PNG)

**Comment by hoppfrosch** [comment]
@cgchoffman : Thanks for the hint - I already have used this. One thing I find annoying with this solution: After restarting Komodo you have to add your directories again ... this it's not a big thing, but simply .... annoying ;-)
